src_types/backend/index.d.ts

```diff
@@ -1,3 +1,2 @@
 /// <reference path="../common/declaration.d.ts" />
-/// <reference path="../common/void$w$w.d.ts" />
-/// <reference path="../common/jsonFiles.d.ts" />
\ No newline at end of file
+/// <reference path="../common/void$w$w.d.ts" />
```

Comment:
consider separating the revert commit

Reply:
That's strange, I did a revert commit!

Reply:
Ohh ok I reverted the removal of that cause I was going to change the implementation

Reply:
So, story is..
I removed this code since I was implementing this in code-code.
Then we decided to stay away from regex's more so I decided to move it BACK to to corvid-types, so I reverted my deletion of this, and this PR changes the implementation of it.